### ISSUE TITLE
Schedule Drag and Resize do not update the event data

### ISSUE THREAD

**Issue by wonbyte** [primefaces/primeng#2070]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**

If you go to http://www.primefaces.org/primeng/#/schedule and drag a few of the existing events to a new time slot. Then change the start date of any of the events and all the moved events will revert to their previous location.

**Expected behavior**

An event should not move location due to interaction with another event.

**Minimal reproduction of the problem with instructions**

Can be reproduced on http://www.primefaces.org/primeng/#/schedule

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **PrimeNG version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

all
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by cagataycivici** [comment]
This seems to be expected as Schedule gets the event array from the angular component and creates a separate internal array of its own with new properties that wrap the event object you've provided. Dragging and Resizing change the start and end dates of this internal event object, not the wrapped object you've provided. Also a callback is provided for you to do changes to the actual event object in events array. If you think, p-schedule component should update the events array which sounds like the angular way, following fixes the issue which gets the actual event and updates start and end dates.

What do you think?

```
eventDrop: (event, delta, revertFunc, jsEvent, ui, view) => {
                let sourceEvent = this.findEvent(event.id);
                if(sourceEvent) {
                    sourceEvent.start = event.start.format();
                    sourceEvent.end = event.end.format();
                }
                
                this.onEventDrop.emit({
                    'event': event,
                    'delta': delta,
                    'revertFunc': revertFunc,
                    'jsEvent': jsEvent,
                    'view': view
                });
            },
```

**Comment by wonbyte** [comment]
@cagataycivici this fix will go with the 2.0.2 release correct? This is how I envision the behavior to work. Thanks for providing the update!

**Comment by cagataycivici** [comment]
2.0.2 yes.
